README.md

```diff
@@ -69,6 +70,16 @@ the [checks documentation page](checks/checks.md).
 
 ## Usage
 
+### Docker
+
+`scorecard` is available as a Docker container:
+
+The `GITHUB_AUTH_TOKEN` has to be set to a valid [token](#Authentication)
+
+``` shell
+docker run -e GITHUB_AUTH_TOKEN=token gcr.io/openssf/scorecard:latest --show-details --repo=https://github.com/ossf/scorecard
+```
+
 ### Using repository URL
 
 The program can run using just one argument, the URL of the repo:
```

Comment:
Unrelated, but could you also update the command line instructions to use either:

```
go get github.com/ossf/scorecard
scorecard --repo=...
```

or 

```
make build-scorecard
./scorecard --repo=...
```

Reply:
I can do that. 👍